src/main/cluster.ts

```diff
@@ -184,6 +184,9 @@ export class Cluster implements ClusterModel {
         this.refreshEvents(),
         this.refreshAllowedResources(),
       ]);
+      if (!this.ready) {
```

Comment:
Shouldn't this have been just `this.ready = true`?

Reply:
Yes...should be fine.

Reply:
It should be. I just saw this as I was going through older PRs to know what was changed. I guess if `ready` was in fact `set ready() { ... }` with a side effect then we may not want to always set it. But I think in this case we can change it later.